src/sage/combinat/sf/character.py

```diff
@@ -151,6 +168,61 @@ def _b_power_k(self, k):
                                     for d in divisors(k))
 
 
+from sage.combinat.sf.sfa import SymmetricFunctionsFunctor
+class CharacterSymmetricFunctionsFunctor(SymmetricFunctionsFunctor):
```

Comment:
Every basis should come with a `_prefix` (since they are all subclasses of CFM). So I don't think this functor is necessary.

Reply:
The strange thing here is that `induced_trivial_character_basis.__init__` takes prefix as a required argument.  Maybe this could or should be made more uniform.  (My previous approach required that the arguments were provided exactly as needed for this `__init__`.)